### ISSUE TITLE
Make whitespace rendered by Refined Github readable

### ISSUE THREAD

**Issue by FichteFoll** [StylishThemes/GitHub-Dark#1050]
The following CSS makes Refined Github's whitespace readable with the dark theme:

```css
.rgh-ws-char::before {
    color: #fff;
    opacity: 0.30;
}
```

Probably requires `!important` because the extension uses exactly the same selector and my temporary custom user style couldn't take priority without it,

Before:
![2020-01-08_22-41-48](https://user-images.githubusercontent.com/931051/72018512-15af0280-3268-11ea-870b-0c7ae1f13e11.png)

After:
![2020-01-08_22-40-30](https://user-images.githubusercontent.com/931051/72018425-e6989100-3267-11ea-8073-afa4c307fb64.png)

**Comment by the-j0k3r** [comment]
> The following CSS makes Refined Github's whitespace readable with the dark theme:

Can you copy and paste the exact rule for that `.rgh-ws-char::before` as its generated by Refined GitHub?

Im not sure about adding options to GitHub Dark that enable extra features for Refined GitHub users, regular users that dont use Refined GitHub would see options that dont apply to them, so makes no sense to add special options that only apply to the few.  None of us actually uses Refined GitHub either but we still try to style their selectors..

>and my temporary custom user style couldn't take priority without it,and my temporary custom user style couldn't take priority without it.

I think you have the right idea by creating a additional style, such things are better to be a part of a single dedicated [Feature Override Style](https://github.com/StylishThemes/Feature-Override-Styles, this is if I understand the request right.

**Comment by silverwind** [comment]
Does this work for you?

```` css
.rgh-ws-char::before {
  color: inherit !important;
}
````

If it does, I'd actually suggest PRing it to RGH and if they don't accept, we can override it. Source is at https://github.com/sindresorhus/refined-github/blob/04dfe9d7cb81cd9b88935a93ec69a40693ee8c36/source/features/show-whitespace.css#L13.

**Comment by FichteFoll** [comment]
`inherit` works, which defers to:

```css
.blob-code, .blob-code-inner {
    color: var(--ghd-code-color, #ccc) !important;
}
```

and with GHD disabled:
```css
.blob-code-inner {
    color: #24292e;
}
```

In fact, I don't see why they even have an explicit color there, so I will PR to remove it. Thanks for the quick support.

**Comment by silverwind** [comment]
Let's keep this open until that PR is merged or rejected.

**Comment by silverwind** [comment]
https://github.com/sindresorhus/refined-github/pull/2688

**Comment by silverwind** [comment]
Fixed in next RGH release.

**Comment by silverwind** [comment]
This is now also fixed on our side as of v1.22.188.
